centrapay/centrapay-docs
# Change Farmlands POS Guide URL

Updated the Farmlands Guide so that it lives at its true canonical URL - `/guides/farmlands-pos-integration`.

This has the side effect of causing the Farmlands Partner Page to be hidden in the navigation but this will be fixed in a follow up PR. For now it is important that Google indexes the correct URL for our customers.

Test plan:
- Confirm that the Farmlands POS guide displays at `/guides/farmlands-pos-integration`
- Confirm that navigating from the Farmlands Partner Page to the POS guide using the Solution button works correctly
